style.css

```diff
@@ -0,0 +1,634 @@
+
+	.main-screen__link::after {
+		content: url('Image_Axion/Vector/Arrow (white 768).png');
+		width: 71px;
+		margin-left: 10px;
+	}
+}
+
+@media (max-width: 376px) {
+	.main-screen {
+		background-image: url(Image_Axion/Photo/rob-potter-422534-unsplash_mobil_375.png);
+		height: 191px;
+		margin: 0 0 38px 0;
+	}
+
+	.main-screen__container {
+		padding: 62px 0 0 48px;
+		width: 180px;
+	}
+
+	.main-screen__title {
+		font-size: 20px;
+		line-height: 19px;
+		margin-bottom: 20px;
+	}
+
+	.main-screen__link {font-size: 12px;}
+
+	.main-screen__link::after {
+		content: url('Image_Axion/Vector/Arrow (1).png');
+		width: 45px;
+		margin-left: 6.29px;
+	}
+}
+/*MAIN section class='goods-list'*/
+
+.goods-list {margin: 0 0 26px 0;}
+
+.content__title {
+	text-align: center;
+	font-size: 22px;
+	font-weight: bold;
+	margin: 0 0 15.26px 0;
+}
+
+.goods {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+
+.goods__item {
+	padding: 0 0 20px 12px;
+	height: 261px;
+	width: 230px;
+	position: relative;
+}
+
+.goods__item:nth-child(n+2)::before {
+	content: url('Image_Axion/Vector/Vector (1).png');
+	display: inline-block;
+	position: absolute;
+	width: 0;
+	height: 101px;
+	border: 1px;
+	margin: 42px 0 0 -30px;
+}
+
+.goods__img {margin: 0 0 28px 10px;}
+
+.goods__name {
+	font-size: 12px;
+	margin: 0 0 3px 0;
+}
+
+.goods__details {
+	display: flex;
+	justify-content: space-between;
+	height: 24px;
+	align-items: center;
+}
+
+.goods__price {font-weight: bold;}
+
+.goods__colors {
+	display: flex;
+	align-items: center;
+}
+
+.goods__color {
+	width: 7px;
+	height: 7px;
+	border-radius: 50%;
+	margin-right: 8px;
+}
+
+.goods__color_active {
+	width: 9px;
+	height: 9px;
+}
+
+.goods__color_grey {background: #C4C4C4;}
+.goods__color_black {background: #0E1211;}
+.goods__color_red {background: #B62E3C;}
+.goods__color_yellow {background: #FDB431;}
+.goods__color_orange {background: #E29D69;}
+.goods__color_blue {background: #257DA9;}
+
+@media (max-width: 769px) {
+	.goods-list {margin: 0 0 78px 0;}
+
+	.content__title {
+		font-size: 30px;
+		margin: 0 0 20px 0;
+	}
+	
+	.goods {padding: 0 70px;}
+
+	.goods__item {
+		height: 352px;
+		width: 281px;
+		padding: 0 0 0px 15px;
+	}
+
+	.goods__item:nth-last-child(1), .goods__item:nth-last-child(2) {display: none;}
+
+	.goods__img {
+		height: 225px;
+		width: 230px;
+		margin: 7px 0 28px 10px;
+	}
+
+	.goods__name {
+		font-size: 20px;
+		margin: 0 0 10px 0;
+	}
+
+	.goods__details {
+		height: 32px;
+		font-size: 25px;
+	}
+
+	.goods__color, .goods__color_active {
+		width: 20px;
+		height: 20px;
+		margin-right: 5px;
+	}
+	
+	.goods__color_grey-changed {order: 3;}
+	.goods__color_black-changed {order: 2;}
+	.goods__color_red-changed {order: 1;}
+	.goods__color_yellow-changed {order: 4;}
+	
+	.goods__item:nth-child(n+2)::before {display: none;}
+}
+
+@media (max-width: 376px) {
+	.goods-list {margin: 0 0 100px 0;}
+
+	.content__title {
+		font-size: 20px;
+		margin: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)